test/data/brave_perf_predictor/entities-httparchive-nostats.json

```diff
@@ -0,0 +1 @@
ame":"eGain","categories":["analytics"],"domains":["analytics-egain.com","egain.com"]},{"name":"TechTarget","categories":["content"],"domains":["techtarget.com","ttgtmedia.com"]},{"name":"Adobe Scene7","company":"Adobe Systems","categories":["content"],"domains":["everestads.net","everestjs.net","scene7.com","wwwimages.adobe.com"]},{"name":"HotelsCombined","categories":["content"],"domains":["datahc.com"]},{"name":"StackAdapt","categories":["ad"],"domains":["stackadapt.com"]},{"name":"The Publisher Desk","categories":["ad"],"domains":["206ads.com","publisherdesk.com"]},{"name":"Ekm Systems","categories":["analytics"],"domains":["ekmpinpoint.co.uk","ekmsecure.com","globalstats.ekmsecure.com"]},{"name":"DistroScale","categories":["ad"],"domains":["jsrdn.com"]},{"name":"Knight Lab","company":"Northwestern University","categories":["utility"],"domains":["knightlab.com"]},{"name":"Vergic Engage Platform","company":"Vergic","categories":["customer-success"],"domains":["vergic.com"]},{"name":"AdCurve","categories":["ad"],"domains":["shop2market.com"]},{"name":"StackExchange","categories":["social"],"domains":["sstatic.net"]},{"name":"MathJax","categories":["utility"],"domains":["mathjax.org"]},{"name":"RebelMouse","categories":["ad"],"domains":["rbl.ms","www.rebelmouse.com"]},{"name":"ShopStorm","categories":["utility"],"domains":["shopstorm.com"]},{"name":"Ad6Media","categories":["ad"],"domains":["ad6media.fr"]},{"name":"OCSP","company":"GoDaddy","categories":["utility"],"domains":["ocsp.godaddy.com","seal.godaddy.com"]},{"name":"Bluecore","categories":["analytics"],"domains":["www.bluecore.com"]},{"name":"Cachefly","categories":["utility"],"domains":["cachefly.net"]},{"name":"Nanorep","company":"Nanorep Technologies","categories":["customer-success"],"domains":["nanorep.com"]},{"name":"AdSpruce","categories":["ad"],"domains":["adspruce.com"]},{"name":"content.ad","categories":["ad"],"domains":["content.ad"]},{"name":"Improve Digital","categories":["ad"],"domains":["360yield.com"]},{"name":"Fastest Forward","categories":["analytics"],"domains":["gaug.es"]},{"name":"RichRelevance","categories":["analytics"],"domains":["richrelevance.com"]},{"name":"ARM","categories":["analytics"],"domains":["tag4arm.com"]},{"name":"Webtrends","categories":["analytics"],"domains":["d1q62gfb8siqnm.cloudfront.net","webtrends.com","webtrendslive.com"]},{"name":"Click4Assistance","categories":["customer-success"],"domains":["click4assistance.co.uk"]},{"name":"Realytics","categories":["analytics"],"domains":["dcniko1cv0rz.cloudfront.net","realytics.net"]},{"name":"Xaxis","homepage":"https://www.xaxis.com/","categories":["ad"],"domains":["t.mookie1.com","247realmedia.com","gmads.net","odr.mookie1.com"]},{"name":"UPS i-parcel","company":"UPS","categories":["other"],"domains":["i-parcel.com"]},{"name":"Qualtrics","categories":["analytics"],"domains":["qualtrics.com"]},{"name":"Adobe Test & Target","company":"Adobe Systems","categories":["analytics"],"domains":["tt.omtrdc.net"]}]
```

Comment:
not sure if we want to have a copy of this cc @AndriusA

Reply:
we could have the config as a component for the full build, but doesn't seem worth it since it doesn't change. There was a problem with android unit tests not finding the file (but working fine in app) - not sure if there is a more robust way of pointing unit tests to the resource file?

Reply:
Added `//brave/components/brave_perf_predictor/resources/entities-httparchive-nostats.json` directly as data file